### PR TITLE
Provide precompiled bundle

### PR DESCRIPTION
**Pull request by Nachtalb**
Fix plone5 resource registry by providing a precompiled bundle

### CODE REVIEW

File: ftw/htmlblock/config.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from Products.CMFPlone.utils import getFSVersionTuple
+
+
+if getFSVersionTuple() > (5, ):
```
Comment (djowett-ftw) [inline ftw/htmlblock/config.py:4]: Something smells wrong here!

File: ftw/htmlblock/upgrades/20191210181147_provide_precompiled_bundle/registry.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<registry>
+
+    <records prefix="plone.resources/htmlblock"
+           interface='Products.CMFPlone.interfaces.IResourceRegistry'
+           remove="True"/>
+
+    <records prefix="plone.bundles/ftw-htmlblock-resources"
+             interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
```
Comment (djowett-ftw) [inline ftw/htmlblock/upgrades/20191210181147_provide_precompiled_bundle/registry.xml:10]: Thinking about it, not sure if we should try 

```
        <value key="resources" purge="true">
        </value>
```
but I've not done it myself yet. @buchi any thoughts on that?

**Review by djowett-ftw — CHANGES_REQUESTED** [review]

File: ftw/htmlblock/config.py (hunk 1 above)
Comment (Nachtalb) [reply ftw/htmlblock/config.py:4]: Fixed

File: ftw/htmlblock/upgrades/20191210181147_provide_precompiled_bundle/registry.xml (hunk 2 above)
Comment (Nachtalb) [reply ftw/htmlblock/upgrades/20191210181147_provide_precompiled_bundle/registry.xml:10]: Done

**Comment by Nachtalb** [comment]
Rebased onto latest master.

**Review by djowett-ftw — APPROVED** [review]
Ok, LGTM!  You could always add these lines to build.js for the future :-)

```
    // Uncomment the line below for JS development
    // optimize: "none",
```

**Pull request merged**